cmd/eksctl-anywhere/cmd/createcluster.go

```diff
@@ -188,3 +191,21 @@ func (cc *createClusterOptions) createCluster(cmd *cobra.Command, _ []string) er
 
 	return createCluster.Run(ctx, clusterSpec, createValidations, cc.forceClean)
 }
+
+func (cc *createClusterOptions) directoriesToMount(clusterSpec *cluster.Spec) []string {
```

Comment:
Maybe I'm wrong but, don't we need this for upgrade as well?

Reply:
We won't because gitops is immutable at the moment, can't change it for upgrade.